src/components/RoomNameInput/index.tsx

```diff
@@ -16,8 +15,7 @@ function RoomNameInput({disabled = false, autoFocus = false, isFocused, value =
     /**
      * Calls the onChangeText callback with a modified room name
      */
-    const setModifiedRoomName = (event: NativeSyntheticEvent<TextInputChangeEventData>) => {
-        const roomName = event.nativeEvent.text;
```

Comment:
The event from live markdown doesn't have `text`, so I need to use `onChangeText` instead. This makes me think that we might get some uncaught errors when replacing all text input with live markdown input, since there is a lot of text input in our app.

Reply:
@ishpaul777 based on the disc, should we just handle the non-text input case?